jekyll.gemspec

```diff
@@ -29,14 +29,15 @@ Gem::Specification.new do |s|
   s.rdoc_options = ["--charset=UTF-8"]
   s.extra_rdoc_files = %w(README.markdown LICENSE)
 
-  s.add_runtime_dependency("liquid",    "~> 3.0")
-  s.add_runtime_dependency("kramdown",  "~> 1.3")
-  s.add_runtime_dependency("mercenary", "~> 0.3.3")
-  s.add_runtime_dependency("safe_yaml", "~> 1.0")
+  s.add_runtime_dependency("addressable", "~> 2.4")
   s.add_runtime_dependency("colorator", "~> 1.0")
-  s.add_runtime_dependency("rouge", "~> 1.7")
   s.add_runtime_dependency("jekyll-sass-converter", "~> 1.0")
   s.add_runtime_dependency("jekyll-watch", "~> 1.1")
+  s.add_runtime_dependency("kramdown",  "~> 1.3")
```

Comment:
`before:` the whitespace after gem-name improved readability

![gemsp](https://cloud.githubusercontent.com/assets/12479464/21791474/ff922f9e-d708-11e6-8a1e-fbd21cf07410.png)

`after:` it just looks weird..

![gemspe](https://cloud.githubusercontent.com/assets/12479464/21791536/772df43e-d709-11e6-8304-2c3cedc53a8d.png)

`suggestion:` **normalize whitespace and improve readability.** :smiley: